.github/ISSUE_TEMPLATE/challenge_proposal.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🚀 Challenge Proposal
+about: I want to add a new challenge!
+title: ''
+labels: challenge
+assignees: ''
+
+---
+
+## Describe the challenge you'd like to add
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+## Suggest a name
+The name of your baby. Consider that there might be adjustment in the challenge name.
+
+## Describe alternatives you've considered
+Mention all the keywords related to the development topics to be address (Adaters, RecyclerView, XML, DI, Architecture, etc..)
+
+## Describe the habilities that we should observe during the challenge resolution
+The idea of the challenges are for interview purposes. We need to ensure that any new challenge provides a way to obtain information about our contestant abilities.
```

Comment:
The idea of the challenges **is that they are used for**... but maybe the whole sentence could be changed to "These challenges should serve interviewing purposes". 

Also, in the second sentence:
"We need to ensure ... about our **contestant's** abilities".